app/scripts/views/complete_sign_up.js

```diff
@@ -44,6 +45,10 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       // cache the email in case we need to attempt to resend the
       // verification link
       this._email = this._account.get('email');
+      this._able = options.able;
+
+      this._experiments = new ExperimentInterface();
+      this._experiments.chooseExperiments();
```

Comment:
Should this use `this._experiments.initExperiment()` instead?

Seems like this would also be appropriate for the mixin's `initialize` method.